generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/errorhandler.interceptor.ts.ejs

```diff
@@ -29,7 +29,7 @@ export class ErrorHandlerInterceptor implements HttpInterceptor {
     intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
         return next.handle(request).pipe(tap((event: HttpEvent<any>) => {}, (err: any) => {
             if (err instanceof HttpErrorResponse) {
-                if (!(err.status === 401 && (err.message === '' || (err.url && err.url.indexOf('/api/account') === 0)))) {
+                if (!(err.status === 401 && (err.message === '' || (err.url && err.url.endsWith('/api/account'))))) {
```

Comment:
When running under a context-path such as `/test`, the path doesn't match as `/api/account` is not in the index 0 spot

Reply:
I think `err.url.includes('/api/account')` might be even better so that if the url ends up having any params it still works